input/fsh/PractitionerSECommonsLite.fsh

```diff
@@ -20,3 +20,75 @@ Description: "This is a light weight Swedish Practitioner profile.
 * telecom.value 1.. MS
 
 Alias: $SEBaseHumanName = http://hl7.se/fhir/ig/base/StructureDefinition/SEBaseHumanName
+
+Instance: minimal-practitioner
+InstanceOf: PractitionerSECommonsLite
+Description: "A simple example of a Swedish Practitioner resource."
+Usage: #example
+* meta.profile = Canonical(PractitionerSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN
+  * system = "urn:oid:1.2.752.29.4.19"
+  * value = "SE2321000016-A123"
+* active = true
+* name
+  * use = #official
+  * family = "Johansson"
+  * given = "Erik"
+  * prefix = "Dr."
+* telecom[0]
+  * system = #phone
+  * value = "+46101234567"
+  * use = #work
+* telecom[+]
+  * system = #email
+  * value = "erik.johansson@example.se"
+  * use = #work
+
+Instance: maximal-practitioner
+InstanceOf: PractitionerSECommonsLite
+Description: "A more thorough example of a Swedish Practitioner resource."
+Usage: #example
+* meta.profile = Canonical(PractitionerSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN "Provider number"
```

Comment:
According to the profile this should be

http://terminology.hl7.org/CodeSystem/v2-0203#PRN

Reply:
Which I know see is solved using aliases :)